sitemedia/js/controllers/annotation_controller.js

```diff
@@ -89,7 +89,10 @@ export default class extends Controller {
 
         // enable annotorious-tahqiq
         const { config, canvasURL, manifestId, annotationContainer } = settings;
-        const anno = Annotorious(viewer);
+        const anno = Annotorious(viewer, { disableDeleteKey: true });
+
+        // get toolbar element
+        const toolbarContainer = this.element.querySelector(".tahqiq-toolbar");
```

Comment:
do you still need to bind this here? what does the annotation controller need to do with it?

Reply:
The annotation controller needs to pass it to tahqiq instantiation, but doesn't need to define a variable for it so I've simplified.